config/appmodel/moduleconfs.data.xml

```diff
@@ -160,14 +185,13 @@
 <obj class="NICReceiverConf" id="def-emu-nic-receiver-conf">
  <attr name="template_for" type="class" val="FDFakeCardReader"/>
  <attr name="emulation_mode" type="bool" val="1"/>
- <attr name="eal_args" type="string" val=""/>
  <rel name="emulation_conf" class="StreamEmulationParameters" id="stream-emu"/>
 </obj>
 
 <obj class="NICReceiverConf" id="def-nic-receiver-conf">
  <attr name="template_for" type="class" val="NICReceiver"/>
  <attr name="emulation_mode" type="bool" val="0"/>
- <attr name="eal_args" type="string" val=""/>
+ <attr name="eal_args" type="string" val="-l 0,1,3,5,7,59,61,63"/>
```

Comment:
The `eal_args` value looks suspiciously like the `used_lcores` in `DPDKPortConfiguration`. Should they be derived from there rather than entered here separately?

Reply:
This is yet another "tuning" artifact in the readout. With kernel isolation and CPU masking EAL does not have visibility on CPU cores to use that are isolated by the kernel. So this is a per-host and pinning related config that now is required by the application itself. Yes, it is very bad and crappy... but I did add it, so the readout app at least can configure. We need to seriously rethink how we want to handle this. This is required by DPDK. We cannot ommit it. Maybe when we have a proper hardware representation in OKS we can generate it. Until then, unfortunately, this is what it is.

Reply:
Also, please mind the difference between `used_lcores` and this string argument list: 0 and 1 are added on top of the used lcores CPU set.